src/shared/variables.css

```diff
@@ -25,4 +25,5 @@
 
   /* Borders */
   --rvr-border-radius: 4px;
+  --rvr-tile-border-radius: 6px;
```

Comment:
This is nitpicky, but I'd rename this one, twice. Here's why:
For any css vars with multiple values (like border-radius, now), the convention should be
* Have an ordered list of the values that doesn't use any semantic names. Think of this like a color palette.
* Then add a semantic section that maps those values to their use cases.

In this case, it would be something like:
```
/* Palette */
--rvr-border-radius-md: 4px;
--rvr-border-radius-lg: 6px;

/* Semantic-ish */
--rvr-border-radius-tile: var(--rvr-border-radius-md);
```

That lets us change the whole feel of the app by changing the "palette" values, but it also gives developers a cheat sheet to know what size we typically use for tile-type components.